slabs.c

```diff
@@ -756,6 +756,11 @@ static int slab_rebalance_start(void) {
         slab_rebal.done = 1;
     }
 
+    //bit-vector to keep track of completed chunks
+    slab_rebal.completed = (uint8_t*)malloc(
+            sizeof(uint8_t)*s_cls->perslab);
```

Comment:
I've been using the calloc pattern in most of the codebase now. It's less error prone than if both malloc/memset lines have to be updated in sync.